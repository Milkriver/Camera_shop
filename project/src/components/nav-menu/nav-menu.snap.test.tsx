import {render} from '@testing-library/react';
import { createMemoryHistory } from 'history';
import HistoryRouter from '../history-router/history-router';
import NavMenu from './nav-menu';

const history = createMemoryHistory();
const mockClassName = '';

describe('Component: NavMenu', () => {
  it('should render correctly', () => {
    const {container} = render(
      <HistoryRouter history={history}>
        <NavMenu classname={mockClassName}/>
      </HistoryRouter>
    );

    expect(container).toMatchSnapshot();
  });
});
