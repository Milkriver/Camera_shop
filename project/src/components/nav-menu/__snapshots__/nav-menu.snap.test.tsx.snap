// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: NavMenu should render correctly 1`] = `
<div>
  <ul
    class="__list"
  >
    <li
      class="__item"
    >
      <a
        class="link"
        href="/catalog.html"
      >
        Каталог
      </a>
    </li>
    <li
      class="__item"
    >
      <a
        class="link"
        href="/"
      >
        Гарантии
      </a>
    </li>
    <li
      class="__item"
    >
      <a
        class="link"
        href="/"
      >
        Доставка
      </a>
    </li>
    <li
      class="__item"
    >
      <a
        class="link"
        href="/"
      >
        О компании
      </a>
    </li>
  </ul>
</div>
`;
